packages/core/upload/admin/src/components/Breadcrumbs/CrumbSimpleMenuAsync.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { NavLink, useLocation } from 'react-router-dom';
+import { useIntl } from 'react-intl';
+
+import { useQueryParams } from '@strapi/helper-plugin';
+import { CrumbSimpleMenu } from '@strapi/design-system/v2/Breadcrumbs';
+import { MenuItem } from '@strapi/design-system/v2/SimpleMenu';
+import { Loader } from '@strapi/design-system/Loader';
+import { useFolderStructure } from '../../hooks/useFolderStructure';
+import { getFolderParents, getFolderURL, getTrad } from '../../utils';
+
+export const CrumbSimpleMenuAsync = ({ parentsToOmit }) => {
+  const [shouldFetch, setShouldFetch] = useState(false);
+  const { data, isLoading } = useFolderStructure({ enabled: shouldFetch });
+  const { pathname } = useLocation();
+  const [{ query }] = useQueryParams();
+  const { formatMessage } = useIntl();
+
+  const allAscendants = data && getFolderParents(data?.[0], query?.folder);
+  const filteredAscendants =
+    allAscendants && allAscendants.filter(ascendant => !parentsToOmit.includes(ascendant.id));
+
+  return (
+    <CrumbSimpleMenu
+      onOpen={() => setShouldFetch(true)}
+      onClose={() => setShouldFetch(false)}
+      aria-label={formatMessage({
+        id: getTrad('header.breadcrumbs.menu.label'),
+        defaultMessage: 'Get more ascendants folders',
+      })}
+      label="..."
+    >
+      {isLoading && (
+        <MenuItem>
+          <Loader small>
+            {formatMessage({
+              id: getTrad('list.asset.load'),
```

Comment:
we also use it anytime we are in a loading situation in the ML, maybe I can change the name of the key to 'content-loading' ?

Reply:
If that's the case then keep it - I'd have suggested to use something like `breadcrumbs.loading`, but I'm fine with keeping it, if you think it's best.

Reply:
after checking I think we would have to create at least 3 different keys with the same value "Loading" or "Content is loading" which I think is better to avoid... but then I agree we should have more generic key naming when we encounter this situation?

Reply:
`"global.isLoading": "Loading"` ?

Reply:
mmh 🤔 the format "global" is only used in the admin/translation so I think global has a different meaning for the admin.
since getTrad already inject a prefix key, here `upload` I guess `content.isLoading` would be enough?
it will be merged with admin as` upload.content.isLoading`, wdyt?